tests/pytests/test_profile.py

```diff
@@ -15,7 +15,7 @@ def testProfileSearch(env):
   conn.execute_command('hset', '1', 't', 'hello')
   conn.execute_command('hset', '2', 't', 'world')
 
-  env.expect('ft.profile', 'profile', 'idx', '*', 'nocontent').error().contains('No `SEARCH` or `AGGREGATE` provided')
+  env.expect('ft.profile', 'profile', 'idx', '*', 'nocontent').error().contains('no such index')
```

Comment:
Previous error message was helpful. Can we keep it?

Reply:
I don't think it was any better to be honest
I see how it is more "basic", but at the end you will get it once you query an existing index so it wasn't removed..
I can add test for it